pennylane_cirq/cirq_device.py

```diff
@@ -34,21 +34,21 @@
 import cirq
 import numpy as np
 import pennylane as qml
-from pennylane import Device
+from pennylane import QubitDevice
 from pennylane.operation import Operation
 
 from ._version import __version__
-from .cirq_interface import CirqOperation, unitary_matrix_gate
+from .cirq_interface import CirqOperation
 
+# pylint: disable=missing-function-docstring
```

Comment:
Pylint complained about the missing docstrings in the Device class, but those are fine as we don't put them for subclasses of `Device` because they are inherited, no?

Now I realize that this might not work here because this is separate from PL. I should copy the docstrings, shouldn't I?

Reply:
Oh, that's odd. When you build the docs, are the inherited docstrings inserted? I wouldn't think you should need to re-write the docstring.

Reply:
It works in the docs 👍